MediaBrowser.Common/Plugins/PluginManifest.cs

```diff
@@ -106,5 +106,12 @@ public PluginManifest()
         /// </summary>
         [JsonPropertyName("imagePath")]
         public string? ImagePath { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether this manifest was created from to the depreciated method of multiple plugins per folder.
+        /// In this instance manifests cannot be stored on disk.
+        /// </summary>
+        [JsonIgnore]
+        public bool Depreciated { get; set; }
```

Comment:
Should this be “Deprecated”?

Reply:
Thanks - Spelling was never my forte!

Reply:
What about the rest of the occurrences?

Reply:
Each occurrence of a dll that contains an IPlugin which are found in the same folder, gets an in memory **only** instance of plugin manifest. The depreciated flag stops these from being written to disk - as they would overwrite each other.

Plugins from elsewhere operate as normal.

Reply:
The spelling error 😉

Reply:
fixed